src/models/quote.js

```diff
@@ -1,17 +1,18 @@
 import Backbone from 'backbone';
 
+
 const Quote = Backbone.Model.extend({
   defaults: {
     symbol: 'UNDEF',
     price: 0.00
   },
-
   buy() {
-    // Implement this function to increase the price by $1.00
+    this.set('price', this.get('price') + 1.00);
+    this.set('buy', true);
```

Comment:
I'm not sure what this accomplishes, quotes just represent the **current** price of a quote, why would there be a buy attribute?